pkg/apis/.import-restrictions

```diff
@@ -0,0 +1,8 @@
+inverseRules:
+  - selectorRegexp: k8s[.]io/kubernetes
+    allowedPrefixes:
+      - ''
```

Comment:
we might want to allow pkg/registry.

Reply:
What happens if we just do that? Probably there is much more, for example admission uses internal types. 

Can you paste the output if we only allow pkg/registry?

We then have to go through that package by package. I expect a number of follow-up cleanup PRs.